tests/test_preprocessing/test_columndropper.py

```diff
@@ -1,6 +1,8 @@
 import pandas as pd
+import polars as pl
```

Comment:
Just some food for thoughts (I can work on this later), we may parametrize the fixture in order to parametrize tests

Reply:
@MarcoGorelli I took the freedom to refactor this test suite via parametrization